hooli-etl-pipeline/README.md

```diff
@@ -0,0 +1,4 @@
+# hooli-etl-pipeline
+
+Based on [Dagster etl tutorial](https://main.archive.dagster-docs.io/etl-pipeline-tutorial/add-a-resource), ported over to run in hooli duckdb locally and Snowflake in branch deployment and prod.
+
```

Comment:
probably worth highlighting the features & benefits here, esp given there's yaml components and assets co-located